gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/analytics/ApiAnalyticsResource.java

```diff
@@ -91,4 +96,18 @@ public ApiAnalyticsAverageConnectionDurationResponse getAverageConnectionDuratio
             .map(ApiAnalyticsMapper.INSTANCE::map)
             .orElseThrow(() -> new NotFoundException("No connection duration found for api: " + apiId));
     }
+
+    @Path("/status-codes-by-entrypoint")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Permissions({ @Permission(value = RolePermission.API_ANALYTICS, acls = { RolePermissionAction.READ }) })
+    public ApiAnalyticsTopHitsByEntrypointResponse getStatusCodesByEntrypoint() {
+        var request = new SearchTopHitsByEntrypointUseCase.Input(apiId, GraviteeContext.getCurrentEnvironment());
+
+        return searchTopHitsByEntrypointUseCase
+            .execute(request)
+            .topHitsAnalyticsByEntrypoint()
+            .map(ApiAnalyticsTopHitsByEntrypointResponse::ofMap)
+            .orElseThrow(() -> new NotFoundException("No data found for api: " + apiId));
```

Comment:
```suggestion
            .orElseThrow(() -> new NotFoundException("No top hits found for api: " + apiId));
```
WDYT ? to keep consistency between others analytics "throw"